mappings/net/minecraft/server/world/ServerWorld.mapping

```diff
@@ -62,18 +68,32 @@ CLASS net/minecraft/class_3218 net/minecraft/server/world/ServerWorld
 	METHOD method_14175 addEntity (Lnet/minecraft/class_1297;)Z
 		ARG 1 entity
 	METHOD method_14176 save (Lnet/minecraft/class_3536;ZZ)V
+		COMMENT Saves the world.
 		ARG 1 progressListener
+			COMMENT the listener for the saving process, or {@code null} to specify none
 		ARG 2 flush
+			COMMENT if it should immediately write all data to storage device
 		ARG 3 savingDisabled
+			COMMENT whether to return early without doing anything
```

Comment:
Does this mean we should always call this method with `false`? If it does nothing, why would one call this method with `true`? (Why does this param exist?)

Reply:
Because :mojank:. (Yes, you should always call this with `false`)

Reply:
Is this still valid after a version bump? Also make sure to check the overrides

Reply:
Still valid, and no class extends ServerWorld.